src/get-started/java-binding.md

```diff
@@ -794,14 +796,15 @@ void auditorTest(@Auditor @ValidatorCount(8) TestKit testKit) {
 
 ### API
 
-To test API implemented with Vertx tools, use the tools described in the
-[project documentation](https://vertx.io/docs/vertx-junit5/java).
-You can use [Vertx Web Client][vertx-web-client] as a client or a different
-HTTP client.
-API can be tested with either unit tests with mocks of services (no TestKit
-required), or with integration tests that use TestKit. For that, TestKit
-provides [`getPort()`][testkit-get-port] method to retrieve a TCP port on
-which the service REST API is mounted for a server to listen on.
+There are two ways to test API. First, API can be tested with unit tests by
+means of service mocks. TestKit is not required for these tests. Second, API
+can be tested with integration tests that use TestKit. For these tests TestKit
+provides the [`getPort()`][testkit-get-port] method that retrieves a TCP port
+to interact with the service REST API.
+
+To test API implemented with Vertx tools, use the tools described in the 
```

Comment:
Checked our unit test in the cryptocurrency, it uses Vertx to create a server via which the API is exposed (and our API requires a Vertx router, so the only alternative to that is using our server (`Server.create()`). The server is needed explicitly only in unit tests, with Testkit it is created under the hood. With either approach, the user can use any HTTP client to send requests.

I'd clarify:

With either approach, you can use any HTTP client to send requests [at your discretion].

To **unit** test API implemented with Vertx tools, use the tools described in the project documentation.